error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -608,7 +618,7 @@ Flux<S> before(
     }
 
     @AfterTemplate
-    Flux<S> after(Mono<T> mono, Function<? super T, ? extends Iterable<? extends S>> function) {
```

Comment:
Hmm, it could make sense, but using `I` instead of `? extends Iterable<? extends S>` would impose a greater restriction than necessary or desired: the rule should also work for any `Iterable` that is not a sub- _or_ super-type of `I`.